_docs/faq.markdown

```diff
@@ -3,17 +3,17 @@ title: FAQ
 order: 999999
 ---
 
-{% include h title="Where is Rust SDK?" tag="sdk" %}
+{% include h title="Where is the Rust SDK?" tag="sdk" %}
 
 We don't use IntelliJ IDEA's SDK concept to manage Rust versions because it is
-specific to IDEA and is not present, for example, in the CLion. Path to Cargo is
-configured per project in the `Settings > Languages & Frameworks > Rust`.
+specific to IDEA and is not present, for example, in CLion. The path to Cargo is
+configured per project through `Settings > Languages & Frameworks > Rust`.
 
-{% include h title="Why Make Project action does not work?" tag="make" %}
+{% include h title="Why does the Make Project action not work?" tag="make" %}
```

Comment:
BTW, this action just works in the latest release, and you can even configure `cargo check` for it :) !